### PR TITLE
fixes scaling

### PR DESCRIPTION
**Pull request by Pokebutt**
fixes the scaling

## Pre-Merge Checklist
- [ ] You tested this on a local server.
- [ ] This code did not runtime during testing.
- [ ] You documented all of your changes.

## Changelog
:cl:
fix: Fixes two-handed wielding smithed weapons doing way too much damage.
/:cl:

### CODE REVIEW

**Comment by Shadow-Quill** [comment]
Checks are failing.

**Comment by Superlagg** [comment]
Checks are still failing. Try building it on your end with an updated branch and see what's breaking.

File: modular_coyote/eris/code/wielding.dm [hunk 1]
```diff
@@ -21,10 +21,9 @@
 	wielded = FALSE
 	if(force_unwielded)
 		force = force_unwielded
-	if(wielded_mult)
-		force = (force / wielded_mult)
 	else
-		force = (force / 1.15)
+		force = (force / wielded_mult)
+		force = (force / FALLBACK_MULT)
```
Comment (Shadow-Quill) [inline modular_coyote/eris/code/wielding.dm:26]: conveniently, the linters mark lines that it's failing on (usually)

Comment (Shadow-Quill) [reply modular_coyote/eris/code/wielding.dm:26]: take a look at the files changed tab

**Comment by Superlagg** [comment]
Are you going to do anything with this?

**Pull request closed without merging**